doc/tutorial/first_steps.md

```diff
@@ -0,0 +1,290 @@
+
+1. Enter the following command to initialise LXD:
+
+       lxd init --minimal
+
+   This will create a minimal setup with default options.
+   If you want to tune the initialization options, see {ref}`initialize` for more information.
+
+## Launch and inspect instances
+
+LXD is image based and can load images from different image servers.
+In this tutorial, we will use the official [`ubuntu:`](https://cloud-images.ubuntu.com/releases/) image server.
+
+You can list all images that are available on this server with:
+
+    lxc image list ubuntu:
+
+See {ref}`images` for more information about the images that LXD uses.
+
+Now, let's start by launching a few instances.
+With *instance*, we mean either a container or a virtual machine.
+See {ref}`containers-and-vms` for information about the difference between the two instance types.
+
+For managing instances, we use the LXD command line client `lxc`.
+See [About `lxd` and `lxc`](lxd-lxc) if you are confused about when to use the `lxc` command and when to use the `lxd` command.
+
+1. Launch a container called "first" using the Ubuntu 22.04 image:
+
+       lxc launch ubuntu:22.04 first
+
+   ```{note}
+   Launching this container takes a few seconds, because the image must be downloaded and unpacked first.
+   ```
+
+1. Launch a container called "second" using the same image:
+
+       lxc launch ubuntu:22.04 second
+
+   ```{note}
+   Launching this container is quicker than launching the first, because the image is already available.
+   ```
+
+1. Copy the first container into a container called "third":
+
+       lxc copy first third
+
+1. Launch a VM using the Ubuntu 22.04 image:
+
+       lxc launch ubuntu:22.04 vm --vm
+
+   ```{note}
+   Even though you are using the same image name to launch the instance, LXD downloads a slightly different image that is compatible with VMs.
+   ```
+
+1. Check the list of instances that you launched:
+
+       lxc list
+
+   You will see that all but the third container are running.
+   This is because you created the third container by copying the first, but you didn't start it.
+
+   You can start the third container with:
+
+       lxc start third
+
+1. Query more information about each instance with:
+
+       lxc info first
+       lxc info second
+       lxc info third
+       lxc info vm
+
+1. We don't need all of these instances for the remainder of the tutorial, so let's clean some of them up:
+
+   1. Stop the second container:
+
+          lxc stop second
+
+   1. Delete the second container:
+
+          lxc delete second
+
+   1. Delete the third container:
+
+          lxc delete third
+
+      Since this container is running, you get an error message that you must stop it first.
+      Alternatively, you can force-delete it:
+
+          lxc delete third --force
+
+   1. Stop and delete the VM:
+
+          lxc stop vm
+          lxc delete vm
+
+See {ref}`instances-create` and {ref}`instances-manage` for more information.
+
```

Comment:
I know `desktop` variants are only available from `images:` but showing something graphical would make a good first impression on the reader IMHO. Maybe the image size and the resource requirements make that not too "introduction friendly" though.

Reply:
Yes, I was considering this, and I don't think the image size would be a problem - but it would mean that we need to go into the different image servers and explain why we're suddenly using a different one. And I think that's outside of the scope of a "first steps" tutorial.